### DISCUSSION TITLE
How to change the clock's timezone?

### DISCUSSION THREAD

**Discussion by Amberwq** [LazyVim/LazyVim#2422] [Q&A] [answered]
The clock in LazyVim is an hour behind on my system. How could I go about fixing this? I'm using a near-default LazyVim, with the only changes being colorscheme and adding Svelte to the formatter

**Comment by dpetka2001** [comment]
Lazyvim doesn't set the time. Your OS is responsible for that.

**Reply by Amberwq** [reply]
Huh, okay. Weird that it's an hour behind then for some reason. But it doesn't really matter so I'll mark this as solved

**Reply by dpetka2001** [reply]
What time does your OS show? Is it different from what Neovim shows?

**Reply by Amberwq** [reply, marked as answer]
Yeah it's different, but Ifigured out the issue. I'm running Neovim inside of my Distrobox container which is apparently an hour behind
